src/utils/print/data.js

```diff
@@ -57,6 +57,29 @@ function stripDatum(d) {
   ));
 }
 
+// eslint-disable-next-line require-jsdoc
```

Comment:
Not sure... These were put in by Jana.  I'm thinking it had to do with not wanting to a have docs auto-generated for them when `npm run apidocs` was run.

They were initially internal functions inside of `selectDailyViewData`, and I'm assuming she didn't want to advertise their use outside of it.

I ended up moving them outside to allow me to export them for proper unit testing.

I don't see any harm in documenting them, though. so I'll just go ahead and do that.